css/main.css

```diff
@@ -0,0 +1,376 @@
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.card-stacks li {
+  background: rgba(255, 255, 255, 0.24) !important;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+}
+
+.card h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.gtime {
+  background-image: url('../media/png/gtime.png');
+}
+
+.weather {
+  background-image: url('../media/png/weather.png');
+}
+
+.conference {
+  background-image: url('../media/png/conference.png');
+}
+
+.katchy {
+  background-image: url('../media/png/katchy.png');
+}
+
+.project-stacks {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: 15px;
+  padding: 20px 0;
+}
+
+.project-stacks li {
+  display: inline;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background: transparent;
+  color: #fff;
+  transition: 350ms;
+  cursor: pointer;
+}
+
+.project-stacks li:hover {
+  background: rgba(0, 0, 0, 0.349);
+}
+
+.about-me,
+.about-me-section {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.about-me-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 12px;
+  background: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: start;
+  transition: 350ms;
+}
+
+.link:hover {
+  background: #a04300;
+}
+
+.about-me-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.list-title {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.languages {
+  list-style: inside url('../media/svg/rectangle.svg');
+}
+
+.frameworks {
+  list-style: inside url('../media/svg/square.svg');
+}
+
+.skills {
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+.contact-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 3rem;
+  padding: 10%;
+}
+
+.contact-intro {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  max-width: 500px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+}
+
+form input {
+  width: 100%;
+  border: none;
+  border-bottom: #dbd8d7 1px solid;
+  padding: 1rem;
+}
+
+form textarea {
+  width: 100%;
+  padding: 1rem;
+  background-color: #fbf8f7;
+}
+
+form button {
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 12px;
+  border: none;
+}
```

Comment:
Good job using the flexbox to position the elements on the homepage! 
However, the large screen version of the webpage doesn't match the Figma template 👇
### Figma design:
![](https://i.imgur.com/Li9Oknm.png)

### Your design:

![](https://i.imgur.com/1aPZEFw.png)

Please make sure to use CSS media queries to display different views for different screen sizes, in order to meet the project requirements!